src/main/java/com/easypost/model/CreditCard.java

```diff
@@ -25,7 +25,7 @@ public static CreditCardFund fund(String amount, Priority primaryOrSecondary) th
     }
 
     /**
-     * Fund a credit card.
+     * Fund your EasyPost wallet by charging your primary or secondary card on file.
```

Comment:
Would have preferred these cleanup items in a separate diff since they don’t have anything to do with the title or summary of the changes in question.

Reply:
I will update the summary since this is a very tiny cleanup.